scripts/query.py

```diff
@@ -0,0 +1,31 @@
+import numpy as np
+
+from qdrant_client import QdrantClient
+
+from qdrant_client.http import models
+
+from openai import OpenAI
+
+
+def main():
+    client = QdrantClient("0.0.0.0", port=6333)
+    openai_client = OpenAI()
+
+    model = "text-embedding-ada-002"
+    prompt = ""
+    embeddings = openai_client.embeddings.create(
+        input = prompt,
+        model=model
+    )
+    tmp = embeddings.data[0].embedding
```

Comment:
Id before a better name here, maybe something like embed, then use that variable below for the query_vector.